src/gcsfilesystem.cpp

```diff
@@ -217,15 +217,15 @@ StatusCode GCSFileSystem::getDirectoryFiles(const std::string& path,
     SPDLOG_TRACE("GCS: listing directory: {}", path);
     auto status = this->getDirectoryContents(path, files);
     if (status != StatusCode::OK) {
-        SPDLOG_WARN("GCS: Unable to list directory content {} -> {}", path,
+        spdlog::error("GCS: Unable to list directory content {} -> {}", path,
             ovms::Status(status).string());
         return status;
     }
     for (auto item = files->begin(); item != files->end();) {
         bool is_directory;
         auto status = this->isDirectory(joinPath({path, *item}), &is_directory);
         if (status != StatusCode::OK) {
-            SPDLOG_WARN("GCS: Unable to list directory content {} -> {}", path,
```

Comment:
Warning.

Reply:
Same with "GCS:" prefix -> we can skip those if we use specialized logger.